hpx/lcos/local/spinlock.hpp

```diff
@@ -15,6 +15,7 @@
 #include <hpx/config.hpp>
 
 #include <hpx/runtime/threads/thread_helpers.hpp>
+#include <hpx/runtime/threads/topology.hpp>
```

Comment:
why did you add this include?

Reply:
Nice catch - it's a leftover. Will remove.

Reply:
Done.